ejabberd/templates/configmap.yaml

```diff
@@ -217,7 +220,20 @@ data:
     sql_database: {{ .Values.postgresql.auth.database }}
     sql_username: {{ .Values.postgresql.auth.username }}
     sql_password: {{ .Values.postgresql.auth.password }}
-    auth_method: [sql]
+    auth_method: [jwt, sql]
```

Comment:
this way the jwt auth gets only enabled if `.Values.postgresql.enabled` is true. So if we disable postgres, no jwt auth